### ISSUE TITLE
Missing maintainer not notified on PR

### ISSUE THREAD

**Issue by adamjstewart** [spack/spackbot#86]
https://github.com/spack/spack/pull/37895 modifies the `py-mdanalysis` package, which @RMeli maintains. However, his review was not requested by spackbot, although many other maintainers were. This seems like a bug in our maintainer detection.

**Comment by RMeli** [comment]
Is there a limit on the numbers of reviewers per PR?

**Comment by adamjstewart** [comment]
Yes. If a PR has [over 100 reviewers](https://github.com/spack/spackbot/blob/05b7bf73bc26fb566f2f317da4adc7b4b6d43249/spackbot/handlers/reviewers.py#L176-L179), spackbot doesn't notify any maintainers. If it has less than 100 reviewers, it should request all reviews. GitHub also has an internal limit, which I believe is also 100.
